front/context/src/AccountsContext.tsx

```diff
@@ -14,6 +14,7 @@ import { Option } from '@polkadot/types';
 import { AccountId, StakingLedger } from '@polkadot/types/interfaces';
```

Comment:
fyi I'm using my own [AccountsContext](https://github.com/paritytech/substrate-light-ui/blob/master/packages/light-apps/src/components/ContextGate/context/AccountContext/AccountContext.tsx) in slui, so feel free to modify this one with breaking changes.

Reply:
ah ok, noted!